camel/loaders/github_loader.py

```diff
@@ -0,0 +1,96 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+
+
+class GitHubLoaderIssue:
+    def __init__(self, title, body, number, file_path, file_content):
+        self.title = title
+        self.body = body
+        self.number = number
+        self.file_path = file_path
+        self.file_content = file_content
+
+    def __eq__(self, other):
+        if isinstance(other, GitHubLoaderIssue):
+            return (
+                self.number == other.number
+                and self.title == other.title
+                and self.body == other.body
+                and self.file_path == other.file_path
+                and self.file_content == other.file_content
+            )
+        return False
+
+    def __repr__(self):
+        return (
+            f"GitHubLoaderIssue(number={self.number}, title={self.title}, "
+            f"body={self.body}, file_path={self.file_path}, file_content={self.file_content})"
+        )
+
+
+class GitHubLoader:
```

Comment:
Should we move the write operation like: `create_pull_request` and `commit_file_content` to the functions?